docs/examples/organizations/consortia_simple.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think this file is no longer referenced? (buyers_suppliers/consortia.json is used instead).

Reply:
You're right. I've removed it in https://github.com/open-contracting/standard/pull/1661/commits/fa9161ae56e8b2e2d0b381d82b1b960b9b9cd484